app/coins/btcsq.js

```diff
@@ -299,15 +299,64 @@ module.exports = {
 		}
 	},
 	blockRewardFunction:function(blockHeight, chain) {
-		var eras = [ new Decimal8(5) ];
-		for (var i = 1; i < 34; i++) {
-			var previous = eras[i - 1];
-			eras.push(new Decimal8(previous).dividedBy(2));
-		}
+		return GetSubsidyMultiplier(blockHeight, chain);
+		// var eras = [ new Decimal8(5) ];
+		// for (var i = 1; i < 34; i++) {
+		// 	var previous = eras[i - 1];
+		// 	eras.push(new Decimal8(previous).dividedBy(2));
+		// }
 
-		var halvingBlockInterval = (chain === "regtest" ? 150 : 1050000);
-		var index = Math.floor(blockHeight / halvingBlockInterval);
+		// var halvingBlockInterval = (chain === "regtest" ? 150 : 1050000);
+		// var index = Math.floor(blockHeight / halvingBlockInterval);
 
-		return eras[index];
+		// return eras[index];
 	}
 };
+
+function GetSubsidyMultiplier(nHeight, chain) {
+	var halvingBlockInterval = (chain === "regtest" ? 150 : 1050000);
+
+	var halvings = (nHeight - 4183200) / halvingBlockInterval;
+	if (halvings >= 64)
+        return 0;
+
+	var nSubsidy = new Decimal8(1504471080 / 1000000000);
```

Comment:
1504471080 / 1000000000 will yield 1.50447108
as I understand it should be 

Decimal8(150447108) 

because precision is 8